app/Enums/RolesEnum.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace App\Enums;
+
+enum RolesEnum: string
+{
+    case ADMIN = 'Admin';
+    case MEMBER = 'Member';
+
+
+    public function label(): string
+    {
+        return match ($this) {
+            static::ADMIN => 'Admin',
+            static::MEMBER => 'Member',
```

Comment:
You can reuse the values of the enums like `ucfirst(self::MEMBER->value)` to make the first letter to be capital.

Reply:
Also use `self::` instead of `static::`